src/Symfony/Component/Messenger/Command/AbstractFailedMessagesCommand.php

```diff
@@ -172,4 +178,26 @@ protected function getLastRedeliveryStampWithException(Envelope $envelope): ?Red
 
         return null;
     }
+
+    private function createCloner(): ?ClonerInterface
+    {
+        if (!class_exists(VarCloner::class) || !class_exists(FlattenException::class)) {
```

Comment:
`class_exists(FlattenException::class)` is not needed, we don't require the implementation here

Reply:
You're right, I removed that check.